README.md

```diff
@@ -1,67 +1,79 @@
+
+## Continuous Integration Build
+
+[![Jenkins Build Status](https://jenkins-2.xenit.eu/buildStatus/icon?job=Xenit+Github%2Falfred-api%2Fmaster&subject=Jenkins)](https://jenkins-2.xenit.eu/job/Xenit%20Github/job/alfred-api/job/master/)
+
+## Publishing
+
+[![Maven Central](https://img.shields.io/maven-central/v/eu.xenit.apix/apix-interface.svg)](http://search.maven.org/#search%7Cga%7C1%7Cg%3A%22eu.xenit.apix%22%20AND%20a%3A%22apix-interface%22)
+
+## Documentation
+
+### Full
+[![Developer guide](https://img.shields.io/badge/Developer_guide-docs.xenit.eu-yellow)](https://docs.xenit.eu/alfred-api)
+
+## Quickstart
+
+### Rules for pull requests
 * Common sense trumps all rules.
-* For every pull request please extend [CHANGELOG.md](./CHANGELOG.md).
+* For every pull request please extend the [CHANGELOG.md](https://github.com/xenit-eu/alfred-api/blob/master/CHANGELOG.md).
 * Do not make breaking changes since this is an API used by customers. Breaking changes include 
   adding, changing or removing endpoints or JSON objects used in requests and responses.
   * If you are forced to make a breaking change:
     * Notify maintainers
     * Add a note to the changelog with upgrade instructions
     * Notify all customers at the next release
 * When working in REST code, please comply to **REST HTTP result codes** policy outlined in the
-  [developer guide](./developer-documentation).
+  [user guide](https://docs.xenit.eu/alfred-api/stable-user).
 * Prefer unit tests over integration tests to keep builds fast
-* Follow our 
-  [coding styleguide and other active procedures](https://xenitsupport.jira.com/wiki/spaces/XEN/pages/624558081/XeniT+Enhancement+Proposals+XEP).
-  * Avoid `this.` prefix for consistency (unless the scope is ambiguous).
-
-
-## Project structure
+  
+### Project structure
 * *apix-interface* builds the interface of Alfred API. This part is agnostic of the 
 Alfresco version used.
 * *apix-rest-v1* builds the REST API of Alfred API. 
 * *apix-impl* builds the AMP which is the main deliverable for Alfred API. The AMP contains the JARs of 
 *apix-interface* and *apix-rest-v1*.
   * The top directory also contains code shared over different Alfresco versions.
-  * *apix-impl/xx* contains all code per Alfresco version. It has a *apix-impl/xx/src/main/java* folder for
-  code specific to that Alfresco version and also includes *apix-impl/src/main/java* folder in its source sets 
-  for the code shared between versions.   
+  * *apix-impl/xx* contains all code per Alfresco version. It has a *src/java* folder
+  for code specific to that Alfresco version and a *src/java-shared code* for the code shared between
+  versions. This code is automatically symlinked from the *apix-impl* directory.   
 * *apix-integrationtests* contains the integration tests for each Alfresco version.
 
+### How to
 
-## How to
-
-### Run
+#### Run
```

Comment:
I think this is a merge that went wrong. This step is gone, this part was removed in master.